pageserver/src/tenant/timeline.rs

```diff
@@ -3850,6 +3850,10 @@ impl Timeline {
     fn set_disk_consistent_lsn(&self, new_value: Lsn) -> bool {
         let old_value = self.disk_consistent_lsn.fetch_max(new_value);
         assert!(new_value >= old_value, "disk_consistent_lsn must be growing monotonously at runtime; current {old_value}, offered {new_value}");
+
+        self.metrics
+            .disk_consistent_lsn_gauge
+            .set(new_value.0 as i64);
```

Comment:
NB that disk_consistent_lsn is kinda meaningless in practice. Not sure if we want to spend a time series per tenant on this.

Reply:
Why? It tells us how much we are lagging by. If it's more than 10GiB. Compute will backpressure.
I want this in order to prove that the remote segment uploads are stalling L0 generation. I expect to see the remote projected lsn be equal to the disk consistent lsn.

Reply:
>  It tells us how much we are lagging by.

What matters is the remote consistent LSN.

> I expect to see the remote projected lsn be equal to the disk consistent lsn.

Yeah. Ok, I don't mind. I think the VictoriaMetrics has enough capacity.